dev-installer

```diff
@@ -21,126 +21,121 @@
 # @author Joseph Perry <joseph@artefactual.com>
 # @version svn: $Id$
 
-cd `dirname $0`
+cd $(dirname "${0}")
 
 part="create the archivematica user"
 echo -n "\"Would you like to ${part}?\" (y/N) "
 read a
-if [[ $a == "Y" || $a == "y" ]]; then
```

Comment:
Why is `=` preferred over `==` inside `[[...]]`? I couldn't find an explanation via Google.